<launch>
    <node name="test_robot_caller" pkg="vlmaps_ros" type="test_robot_caller.py" output="screen"></node>
    <group ns="vlmaps_robot">
        <rosparam file="$(find vlmaps_ros)/config/vlmaps_robot.yaml" />
    </group>
</launch>